src/components/Delta.tsx

```diff
@@ -42,35 +42,36 @@ function Delta({
     </Text>
   ) : null;
 
+  const delta =
+    percent && valueChange.percentage
+      ? valueChange.percentage * 100
+      : valueChange.value;
+
+  const [color, ArrowIcon, sign] =
+    delta !== 0
+      ? delta > 0
+        ? ["success.c100", ArrowUpMedium, "+"]
+        : ["error.c100", ArrowDownMedium, "-"]
+      : ["neutral.c100", () => null, ""];
+
   if (
     percent &&
     ((valueChange.percentage === 0 && !show0Delta) ||
       valueChange.percentage === null ||
       valueChange.percentage === undefined)
   ) {
     if (fallbackToPercentPlaceholder) return percentPlaceholder;
+    if (percent && ArrowIcon) return <ArrowIcon size={16} color={color} />;
```

Comment:
`!!ArrowIcon` is always true no ? even if it's `() => null`

Reply:
yes, I kept the logic below at line 77 in prevention for further modifications where we would not put a function but just a null value.
But currently yet it's useless I can remove it if you want